docs/whatsnew.rst

```diff
@@ -93,82 +93,56 @@ space's name wiki".
 Technical novelties in eXo Platform 5.1
 ========================================
 
-**Upgrades**
+**New features**
 
-In the version 5.0, many tools/applications used by eXo Platform were 
-upgraded to newer  versions in order to benefit from their latest 
-updates and improvements: 
+In the version 5.1, we integrated a new feature which is the push notifications on mible.
+This new feature is supported by eXo mobile applications. It relies on Firebase Cloud Messaging 
+and a new addon packaged by default in the Enterprise Edition (exo-push-notifications).
 
-- Tomcat 8.5
+**Upgardes**
 
-- JBoss EAP 7.0
+In 5.1 version, eXo Platform upgraded to Jboss EAP 7.1 in order to:
 
-- Elasticseach 5.6 
+- Benefit from its lastest improvements such as:
 
-- Infinispan 8
+  - Use the new security framework WildFly Elytron
 
-- Groovy 2.4
+  - Support for HTTP/2
 
-- JQuery 3.2 
+- Fix some issues occuring with the previous version:
 
-- Shindig 2.5
+  - "Remember me" token not recognized in Jboss EAP 7.0
+  
+  - Instance id of a cluster node is not added on the JSESSIONID cookie
 
 **Performances and scalability**
 
-In eXo Platform 5.0, we insisted on improving the performance and the 
+In eXo Platform 5.1, we insised on improving the performance and the 
 scalability by changing some strategies, configurations and data 
 storage.  
-
-- Cache strategy and configuration
-
-- New cache on organization service
-
-- Files indexing in Elasticsearch
-
-- Storage of all spaces members in Social database tables
-
+- Users/Groups/Roles import in database: a new job is implmentd to import IDM data 
+  in IDM database asynchronously and periodically.
+  
+- Smarter data user synchronization: only mandatory data is synchronized, other data 
+is created when the user authenticates to the platform.
+  
 **Migrations**
 
-With eXo Platform 5.0, some data were migrated from JCR to database to 
-improve performance and scalability:
-
-- Notifications and settings in JPA dataset
-
-- ECMS files are indexed in Elasticsearch
-
-- Space members stored in Social database tables 
-
-Many other improvements and refinements were also done in :
-
-- Customizing skin: No more need to duplicate the whole project 
-  platform-ui.
-
-- Extending the IM list of user profile is now doable through UI.
-
-- When indexing JCR workspace, old indexes are always existing until the 
-  end of indexation.
-
-- An Mbean is added to reload Javascript.
-
-- A new minify attribute allowing to enable/disable the minification.
-
-- Compilation of Javascript scripts by Google Closure Compiler to ES5.
+As cited in the previous section, eXo Chat has a new design. 
+This new design is powered by the frontend framework Vue.js.
+Following this framework change, Chat favorites data has also been changed. An upgrade plugin runs 
```

Comment:
The Chat favorites data model change is not directly related to the new frontend framework. We can talk of both topics (frontend framework change, and favorites migration), but without linking them together.